pennylane/ops/op_math/controlled_ops.py

```diff
@@ -144,7 +147,12 @@ def _primitive_bind_call(
             warnings.warn(
                 "base operator already has wires; values specified through wires kwarg will be ignored."
             )
-            # wires = Wires(())
+            warnings.warn(
+                "QubitUnitary input to ControlledQubitUnitary is deprecated and will be removed in v0.42. "
+                "Instead, please use a full matrix as input.",
+                qml.PennyLaneDeprecationWarning,
```

Comment:
Maybe we should suggest `qml.ctrl` as an alternative to create a controlled `QubitUnitary`